pipelines/tagger_parser_ud/project.yml

```diff
@@ -84,6 +84,6 @@ commands:
   - name: clean
     help: "Remove intermediate files"
     script:
-      - "rm -rf training/*"
-      - "rm -rf metrics/*"
-      - "rm -rf corpus/*"
+      - "python utils/file.py rm training/*"
+      - "python utils/file.py rm metrics/*"
+      - "python utils/file.py rm corpus/*"
```

Comment:
Here, we could just have a custom Python script that cleans the correct output directories recursively.